packages/account/src/Sections/Security/ConnectedApps/template-helper.tsx

```diff
@@ -1,34 +1,24 @@
 import React from 'react';
-import { localize } from '@deriv/translations';
+import { Localize, localize } from '@deriv/translations';
+
+export const CONNECTED_APPS_COLUMN_NAMES = [
```

Comment:
Localize should be inside a function so it'll be called again after a language switch, as suggested by @balakrishna-deriv that coz it's a constant, it won't render again within the lifecycle

Reply:
on changing the language, the page is getting refreshed and therefore the language is getting updated accordingly. so translation in connected apps is working fine this way.
also @likhith-deriv confirmed that `Localize` component rerenders even when used from array.